### PR TITLE
[ID] Cleanup `Global Moderation Team`

### PR DESCRIPTION
**Pull request by Nivalyx**
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 1]
```diff
@@ -10,51 +10,51 @@ tags:
...
+ - perilaku yang buruk pada forum (cth. postingan yang tidak pantas)
+ - diskusi dan kolom komentar beatmap
+ - moderasi profil pengguna (terutama yang mencakup konten NSFW atau eksplisit pada avatar, sampul atau detail profil, dsb.)
 
 ## Menjadi seorang moderator
 
-Menjadi anggota dari Global Moderation Team membutuhkan jumlah kontribusi yang signifikan. Calon anggota sering diperhatikan dari kerelaan mereka untuk membantu pengguna lain, aktivitas di dalam komunitas, dan berkelakuan baik. Kemampuan mereka untuk berbicara bahasa kedua selain bahasa Inggris sangat disukai. Kandidat didiskusikan di dalam tim untuk menentukan penambahan potensial sebagai member tim.
+Keanggotaan Global Moderation Team memerlukan kontribusi yang jumlahnya tidak sedikit. Para calon anggota pada umumnya dinilai berdasarkan kerelaan mereka untuk membantu pengguna lain, aktivitas mereka di dalam komunitas, dan riwayat kelakuan baik mereka. Kemampuan untuk berbicara bahasa lain di samping bahasa Inggris merupakan nilai tambah. Mereka yang terpilih selanjutnya akan didiskusikan secara internal untuk menentukan apakah mereka layak untuk diangkat sebagai moderator.
```
Comment (wowcake) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:31]:
```suggestion
Keanggotaan Global Moderation Team memerlukan kontribusi yang jumlahnya tidak sedikit. Para calon anggota pada umumnya dinilai berdasarkan kerelaan mereka untuk membantu pengguna lain, aktivitas mereka di dalam komunitas, dan riwayat perilaku baik mereka. Kemampuan untuk berbicara bahasa lain di samping bahasa Inggris merupakan nilai tambah. Mereka yang terpilih selanjutnya akan didiskusikan secara internal untuk menentukan apakah mereka layak untuk diangkat sebagai moderator.
```

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 2]
```diff
@@ -10,51 +10,51 @@ tags:
...
 
-### Bagaimana cara agar dipertimbangkan
+### Cara untuk dapat dipertimbangkan menjadi moderator
 
-Faktor terpenting agar dipertimbangkan adalah perilaku, aktivitas, pemahaman terhadap [peraturan](/wiki/Rules), dan kesabaran:
+Agar kamu dapat dipertimbangkan menjadi moderator, faktor terpenting yang harus kamu usung antara lain adalah perilaku, aktivitas, pemahaman terhadap [peraturan](/wiki/Rules), dan kesabaran:
```
Comment (wowcake) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:35]:
```suggestion
Agar kamu dapat dipertimbangkan menjadi moderator, faktor terpenting yang harus kamu miliki antara lain adalah perilaku, aktivitas, pemahaman terhadap [peraturan](/wiki/Rules), dan kesabaran:
```

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 3]
```diff
@@ -10,51 +10,51 @@ tags:
...
-2. Jangan melaporkan setiap hal kecil dalam obrolan. Itu adalah kemampuan yang penting untuk mengetahui apa yang layak dilaporkan dan apa yang tidak.
-3. Jangan menjawab pertanyaan orang lain tanpa mengetahui apakah jawaban tersebut benar atau tidak, khususnya di forum. Saran diperbolehkan, tetapi orang yang memberi informasi yang salah tidak akan membantu siapa pun. Jika Kamu tidak mengetahui solusi dari suatu masalah, biarkan orang lain yang menjawab dan belajarlah dari mereka.
-4. Jangan berlebihan. Tim mencari kontribusi yang konsisten, bukan kontribusi yang jumlahnya berlebihan. Hal ini diketahui dapat membuat orang marah dan kehilangan kesabaran dan komitmen. Santai saja!
+1. Jangan bertanya secara terus-menerus kepada GMT tentang apakah kamu berada dalam daftar kandidat yang berpotensi atau tidak. GMT tentunya tidak ingin menambahkan anggota yang berkontribusi semata-mata hanya untuk memperoleh gelar. Berpikir untuk kebaikan sesama adalah kuncinya.
```
Comment (wowcake) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:46]:
```suggestion
1. Jangan bertanya secara terus-menerus kepada GMT tentang apakah kamu berada dalam daftar kandidat yang berpotensi atau tidak. GMT tentunya tidak ingin menambahkan anggota yang berkontribusi semata-mata hanya untuk memperoleh gelar. Mempunyai pola pikir untuk kebaikan sesama adalah kuncinya.
```

**Comment by Nivalyx** [comment]
applied everything, ty for the review as always @wowcake 🦉

**Review by wowcake — APPROVED** [review]
npnp 👍 great work as always, good to go

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**